pages/sites.tsx

```diff
@@ -19,10 +36,9 @@ const Sites = () => {
           )}
         </button>
       </div>
-      <SiteCardLink isEditMode={editMode} />
-      <SiteCardLink isEditMode={editMode} />
-      <SiteCardLink isEditMode={editMode} />
-      <SiteCardLink isEditMode={editMode} />
+      {siteUUIDs.map((site_uuid: string, idx: number) => (
+        <SiteCardLink key={idx} siteUUID={site_uuid} isEditMode={editMode} />
+      ))}
```

Comment:
great catch to put `key={idx}` in, and this definitely works, but this can lead to things breaking if we aren't careful
 
the `key` property is meant to allow React to keep track of distinct objects. when using the index of each element in the list as a key, if the underlying list of `siteUUIDs` changes, React might easily lose track. You can find specific examples of why this might cause error [in this article](https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/). 

so instead, the `key` should be something more stable, and more unique to each element within `siteUUIDs`. a better option would naturally then be the `siteUUID` itself, as this would definitely be more permanent and unique. 

```suggestion
      {siteUUIDs.map((site_uuid: string) => (
        <SiteCardLink key={site_uuid} siteUUID={site_uuid} isEditMode={editMode} />
      ))}
```